mlos_bench/mlos_bench/storage/sql/experiment.py

```diff
@@ -125,6 +128,39 @@ def _setup(self) -> None:
                         exp_info.git_commit,
                     )
 
+    def save(self) -> None:
+        self._ensure_persisted()
+
+    def _setup(self) -> None:
+        super()._setup()
+        self._ensure_persisted()
+        with self._engine.begin() as conn:
```

Comment:
Might need to separate that out to an `_update` method, or just incorporate it into `_save`

Reply:
Could also rename `_save` to `_create` to separate the `INSERT` from `UPDATE` calls